Validation/RecoEgamma/plugins/ElectronMcFakeValidator.h

```diff
@@ -67,7 +67,7 @@ class ElectronMcFakeValidator : public ElectronDqmAnalyzerBase {
 
   double maxPt_;
   double maxAbsEta_;
-  double deltaR_;
+  double deltaR_, deltaR2_;
```

Comment:
At this point. deltaR_ is not needed any more as class member, and it can be removed from them

Reply:
OK, deltaR_ is removed. Please note that we have planned a PR during november/december in order to do such a modifications.